test/gtest/ucp/ucp_test.h

```diff
@@ -46,8 +48,11 @@ class ucp_test_base : public ucs::test_base {
         typedef std::vector<ucs::handle<ucp_ep_h, entity *> > ep_vec_t;
         typedef std::vector<std::pair<ucs::handle<ucp_worker_h>,
                                       ep_vec_t> > worker_vec_t;
+        typedef std::deque<std::pair<ucp_ep_h, void *> > closing_eps_t;
```

Comment:
why do we need to keep ucp_ep_h here? looks like it's never used
if we have only the request, we could test for it directly and will not need closing_ep_t